plugins/SegmentEditor/SegmentSelectorControl.php

```diff
@@ -54,8 +54,7 @@ public function __construct($idSite = false)
             if ($segment['category'] == $visitTitle
                 && ($segment['type'] == 'metric' && $segment['segment'] != 'visitIp')
             ) {
-                $metricsLabel = Piwik::translate('General_Metrics');
-                $metricsLabel[0] = Common::mb_strtolower($metricsLabel[0]);
+                $metricsLabel = Common::mb_strtolower(Piwik::translate('General_Metrics'));
```

Comment:
It's the real fix actually, we're now lowercasing the whole string which has the same effect of just lower casing the first letter (in most languages anyway)